app/src/features/surveys/observations/sampling-sites/SamplingSiteList.tsx

```diff
@@ -64,6 +68,65 @@ const SamplingSiteList = () => {
     setSelectedSampleSiteId(sample_site_id);
   };
 
+  /**
+   * Handle the delete sampling site API call.
+   *
+   */
+  const handleDeleteSampleSite = async () => {
+    await biohubApi.samplingSite
+      .deleteSampleSite(surveyContext.projectId, surveyContext.surveyId, selectedSampleSiteId as number)
```

Comment:
Should we convert this to a number (`Number(selectedSampleSiteId)`) rather than cast? (Just to be safe)